README.md

```diff
@@ -165,22 +165,31 @@ import (
 )
 
 func Handle(w http.ResponseWriter, r *http.Request) {
-    // read request payload
-	reqBody, err := ioutil.ReadAll(r.Body)
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-		return
+	var input []byte
+
+	if r.Body != nil {
+		defer r.Body.Close()
+
+		// read request payload
+		reqBody, err := ioutil.ReadAll(r.Body)
+
+		if err != nil {
```

Comment:
If we ignore body read errors we can keep it more succinct

Reply:
Thanks, I addressed your suggestion :+1: